src/nectarchain/dqm/pixel_timeline.py

```diff
@@ -0,0 +1,155 @@
+from dqm_summary_processor import DQMSummary
+from matplotlib import pyplot as plt
+from ctapipe.visualization import CameraDisplay
+from ctapipe.instrument import CameraGeometry
+from ctapipe.coordinates import EngineeringCameraFrame
+import numpy as np
+
+
+class PixelTimeline_HighLowGain(DQMSummary):
+    def __init__(self, gaink):
+        self.k = gaink
+        return None
+
+    def ConfigureForRun(self, path, Pix, Samp, Reader1):
+        # define number of pixels and samples
+        self.Pix = Pix
```

Comment:
Duplicated code fragment (12 lines long)

Reply:
Better would have to have a common `ConfigureForRun` method in a base class, which would be inherited and called within `pixel_timeline.py` and `pixel_participation.py`.